debian/config/config.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+start() {
+  if [ -r /etc/default/verify ]; then
+    . /etc/default/verify
```

Comment:
`/etc/default/verify` is no longer a thing and can be removed, but it won't do any harm so it won't stop me approving this